tests/kohana/Log/WriterTest.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+/**
+ * Tests Kohana Logging API - writing to Syslog
+ *
+ * @group kohana
+ * @group kohana.core
+ * @group kohana.core.logging
+ * @group kohana.core.logging.syslog
+ *
+ * @package    Kohana
+ * @category   Tests
+ * @author     Kohana Team
+ * @copyright  (c) 2008-2012 Kohana Team
+ * @license    http://kohanaframework.org/license
+ */
+class Kohana_Log_WriterTest extends Unittest_TestCase
```

Comment:
It might be better to make this a base class and extend the other writer tests from it? That might reduce some of the duplication of filtering tests, and ensure that they're all doing eg format as expected - I think that's better than explicitly testing the methods of an abstract class.